api/namex/models/request.py

```diff
@@ -619,7 +619,9 @@ def on_insert_or_update_nr(mapper, connection, request):
 
        Temporary NRs (nrNum starting with 'NR L') are discarded.
     """
-    if current_app.config.get('DISABLE_NAMEREQUEST_NATS_UPDATES', 0) != 1 and not request.nrNum.startswith('NR L'):
+
+    if current_app.config.get('DISABLE_NAMEREQUEST_NATS_UPDATES', 0) != 1 and not request.nrNum.startswith('NR L') \
+        and request.entity_type_cd != 'FR' and request.entity_type_cd != 'GP':
```

Comment:
You can use _in_ and break down the conditional a bit
```suggestion
    if current_app.config.get('DISABLE_NAMEREQUEST_NATS_UPDATES', 0) != 1 \
       and not request.nrNum.startswith('NR L') \
       and request.entity_type_cd not in ('FR', 'GP'):
```

Reply:
thank you for suggestion, changed, and repushed.